src/components/Alert.react.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Store from '../store'
+import { connect } from 'react-redux'
+import AlertActions from '../actions/alerts'
+
+class Alert extends React.Component {
+
+  render() {
+    const alert = this.props.alert;
+    return !alert ? <div/> :
+      <div>
+        <p>{alert.message} <span onClick={this._cleanAlert}>x</span>
+        </p>
+      </div>
+  }
+
+  _cleanAlert = e => {
+    e.preventDefault();
+      Store.dispatch(AlertActions.reset());
```

Comment:
You shouldn't need to be calling this directly from the `Store`, just use `mapDispatchToProps` to register the reset action within this component `props`.

Reply:
That looks good. We would need to change it everywhere, and as this is already too big and mostly copying the style from basil, I would prefer to do it next. I reported this bug: https://github.com/TPL-protocol/demo-dapp/issues/3

Reply:
Indeed, it's just a react-redux best practice; no need to actually change it